### PR TITLE
Add receiver to udev rules

### LINKED ISSUE #125: Some receivers expected by Solaar GUI are not listed in udev rules file
Besides my M525, I also have an old V220 wireless mouse. It comes with a USB receiver that shows in `lsusb` as:

`ID 046d:c51b Logitech, Inc. V220 Cordless Optical Mouse for Notebooks`

This is considered as a Logitech Receiver by Solaar (as `NANO_RECEIVER_C51B` in `lib/logitech_receiver/base_usb.py`), but this ID is not present in the udev rules file. 

As a consequence, every time Solaar starts I get this annoying alert box:

_Found a Logitech Receiver (/dev/hidraw3), but did not have permission to open it.
If you've just installed Solaar, try removing the receiver and plugging it back in._

If this receptor is supported by Solaar, it should be present in the udev rules file. If it is not supported, then the "permissions error" alert should not be displayed for this receiver (since removing/plugging it back won't help at all).

### PR DESCRIPTION
**Pull request by jrbenito**
Fixes #125

Signed-off-by: Josenivaldo Benito Jr <jrbenito@benito.qsl.br>

### CODE REVIEW

**Comment by jrbenito** [comment]
There are other IDs not inserted on udev rules. Shall I add them too?

**Comment by Lekensteyn** [comment]
Can you add the device (V220 wireless mouse) in the comments?

And yes, some files have gotten out of sync, it would be nice if they could be updated

**Comment by jrbenito** [comment]
@Lekensteyn 

Commented as you suggested and added other PIDs present on `lib/logitech_receiver/base_usb.py` to the `udev` rules, this synchronizes both.

File: rules.d/42-logitech-unify-permissions.rules [hunk 1]
```diff
@@ -20,9 +20,20 @@ ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c526", GOTO="solaar_apply"
...
 # G-Series receiver -- G-Series mouse
 ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c531", GOTO="solaar_apply"
 
+# other nano receivers Solaar known
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c517", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c518", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c51a", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c521", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c525", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c526", GOTO="solaar_apply"
```
Comment (Lekensteyn) [inline rules.d/42-logitech-unify-permissions.rules]: c526 is already known (see above)

what is "na receiver"?

Comment (jrbenito) [inline rules.d/42-logitech-unify-permissions.rules]: > what is "na receiver"?

It´s a typo, should be nano. 

> c526 is already known (see above)
keen eye, thanks!

Comment (jrbenito) [inline rules.d/42-logitech-unify-permissions.rules]: Done

File: packaging/debian/solaar.udev [hunk 2]
```diff
@@ -20,9 +20,20 @@ ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c526", GOTO="solaar_apply"
...
 # G-Series receiver -- G-Series mouse
 ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c531", GOTO="solaar_apply"
 
+# other Nano receivers Solaar known
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c517", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c518", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c51a", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c521", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c525", GOTO="solaar_apply"
+ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c526", GOTO="solaar_apply"
```
Comment (jrbenito) [inline packaging/debian/solaar.udev]: Sure!! Thanks

Comment (Lekensteyn) [inline packaging/debian/solaar.udev]: The duplicate c526 line still exists :>

**Pull request merged**